iceoryx_utils/include/iceoryx_utils/log/logger.hpp

```diff
@@ -44,6 +44,7 @@ class Logger
 
     void SetLogLevel(const LogLevel logLevel) noexcept;
     void SetLogMode(const LogMode logMode) noexcept;
+    bool IsEnabled(const LogLevel logLevel) const noexcept;
```

Comment:
Wrong Code Style. Aren't we using lowerCamelCase?! I think we have to adjust this.

Reply:
This was based on the ARA API, that's the reason for this coding style